variables.tf

```diff
@@ -530,6 +530,36 @@ variable "sku_tier" {
   default     = "Free"
 }
 
+variable "storage_profile_blob_driver_enabled" {
+  type        = bool
+  description = "(Optional) Is the Blob CSI driver enabled? Defaults to false"
+  default     = null
```

Comment:
According to the [schema](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/internal/services/containers/kubernetes_cluster_resource.go#L1064-L1068), the `default` should be `false`.

Reply:
Changed